ebos/ecltracermodel.hh

```diff
@@ -31,8 +31,13 @@
 #include <ebos/eclgenerictracermodel.hh>
 
 #include <opm/models/utils/propertysystem.hh>
+
 #include <opm/simulators/utils/VectorVectorDataHandle.hpp>
 
+#include <array>
+#include <iostream>
```

Comment:
I know it's needed here, but we really should work to avoid including `<iostream>` in our headers.